src/oic/oic/message.py

```diff
@@ -727,19 +749,44 @@ def verify(self, **kwargs):
 
 
 class RefreshSessionRequest(Message):
-    c_param = {"id_token": SINGLE_REQUIRED_STRING,
-               "redirect_url": SINGLE_REQUIRED_STRING,
-               "state": SINGLE_REQUIRED_STRING}
+    c_param = {
+        "id_token": SINGLE_REQUIRED_STRING,
+        "redirect_url": SINGLE_REQUIRED_STRING,
+        "state": SINGLE_REQUIRED_STRING
+    }
+
+    def verify(self, **kwargs):
+        super(RefreshSessionRequest, self).verify(**kwargs)
```

Comment:
`super().verify(**kwargs)` is sufficient. This is on multiple places.

Reply:
Nothing new added by me. But while cleaning up ...

Reply:
The same as above... it appeared as a new code, hence the comment :)